src/app/application.cpp

```diff
@@ -61,7 +61,7 @@
 #endif // Q_OS_MACOS
 #endif
 
-#include "base/bittorrent/infohash.h"
+#include "base/bittorrent/sha1hash.h"
```

Comment:
Wrong order now. (in other files too)

Reply:
Another idea is to keep the previous file name (infohash.\*) and put SHA256Hash there as well (when we need it). Or would it be better to have them in separate files?

Reply:
>Another idea is to keep the previous file name (infohash.*) and put SHA256Hash there as well (when we need it).

We can really turn it to template and instantiate it for SHA1 and SHA256 hashes (similar to libtorrent).

Reply:
>We can really turn it to template and instantiate it for SHA1 and SHA256 hashes (similar to libtorrent).

I don't mind that if most of the function code can be shared across.

Reply:
I'll think about it later anyway. For now, I want to leave everything as it is (SHA1 only). So the question is should I return the previous file name, or leave the new one and fix the order of header inclusions?

Reply:
>So the question is should I return the previous file name, or leave the new one and fix the order of header inclusions?

If you are going to template it then `infohash.*` is acceptable for me, otherwise I would prefer some specific name such as `sha1hash.*`.

Reply:
>If you are going to template it then `infohash.*` is acceptable for me,

Well, done.